docs/backend/auth/keycloak.rst

```diff
@@ -76,6 +77,107 @@ Interaction schema
 Basic configuration
 -------------------
 
-.. autopydantic_model:: syncmaster.settings.auth.keycloak.KeycloakProviderSettings
-.. autopydantic_model:: syncmaster.settings.auth.jwt.JWTSettings
+.. autopydantic_model:: syncmaster.backend.settings.auth.keycloak.KeycloakAuthProviderSettings
+.. autopydantic_model:: syncmaster.backend.settings.auth.keycloak.KeycloakSettings
+.. autopydantic_model:: syncmaster.backend.settings.auth.jwt.JWTSettings
+
+
+Local installation and testing
+-----------------------------
+
+You can test Keycloak auth locally with docker compose:
+
+
+.. code-block:: console
+
+    $ docker compose -f docker-compose.test.yml up keycloak -d
+
+
+At first, you have to go to `http://localhost:8080/admin <http://localhost:8080/admin>`_ and login via login: `admin`, password: `admin` (by default) to create realms.
+
+.. image:: images/keycloak-login.png
+   :width: 400px
+   :align: center
+
+
+Create new realm:
```

Comment:
IMHO it's better to replace:
```
Make something:

.. image

text
```

with:
```
Make something
~~~~~~~~~~~~~~

.. image

text
```

As it allows to navigate through headers on the right side of a page.